src/Tests/Essential.Diagnostics.SeqTraceListener.Tests/SeqStructuredDataTests.cs

```diff
@@ -470,6 +471,7 @@ public void SeqStructuredCustomObject()
         [TestMethod]
         public void SeqDestructureCustomObject()
         {
+            Thread.CurrentThread.CurrentCulture = new CultureInfo("en-US");
```

Comment:
I know this is only a test, but why is this here?

If you need to do a culture-specific test, then create a CurrentCultureScope : IDisposable that saves the current culture, changes the value, then changes it back on Dispose(), to preserve whatever was there before.

I presume this is because decimal handling in some countries is "1,2" instead of "1.2".

If so, this fix should be a separate pull request from adding the error handling flag.

Also, it is a good question whether this should be fixed by setting an explicit culture scope on the test, i.e. it should destructure based on string representation in the current culture, or if it is a bug and should be destructuring to a JSON-compatible format. 

i.e. numerics should be output as "1.2" irrespective of culture, as it is a data format (not a string output).

Add a separate pull request and I will look at the expectations (i.e. should we be fixing the test, or the code).

Reply:
I added this so i could run the tests before i would commit it. (I will take a look next week if you find it necessary to be fixed)